lighthouse-core/report/v2/report-renderer.js

```diff
@@ -81,36 +104,93 @@ window.ReportRenderer = class ReportRenderer {
    * @param {!Object<string, string>=} attrs Attribute key/val pairs.
    * @return {!Element}
    */
-  _createElement(name, className, attrs = {}) {
+  createElement(name, className, attrs = {}) {
     const element = this._document.createElement(name);
     if (className) {
       element.className = className;
     }
-    for (const [key, val] of Object.entries(attrs)) {
-      element.setAttribute(key, val);
-    }
+    Object.keys(attrs).forEach(key => {
+      element.setAttribute(key, attrs[key]);
+    });
     return element;
   }
 
   /**
-   * @param {number} score
-   * @param {string} title
-   * @param {string} description
+   * @param {!string} selector
+   * @return {!DocumentFragment} A clone of the template content.
```

Comment:
TBH, it's a terrible API and name. `importNode()` returns the node you import. Since we're importing a `DocumentFragment`, we get back a `Node` of that type. 

![screen shot 2017-04-06 at 12 35 37 pm](https://cloud.githubusercontent.com/assets/238208/24772285/8e68f978-1ac5-11e7-85d9-923aa5da73a1.png)